### PR TITLE
Add Asus TUF FX504GD

### PR DESCRIPTION
**Pull request by ImExtends**
Request to add Asus TUF FX504GD to nixos-hardware channel.
Nouveau doesn't work at all on this laptop, so I add nouveau.modeset=0 as a kernel param.

### CODE REVIEW

**Comment by jonringer** [comment]
I would squash the commits

**Comment by ImExtends** [comment]
> I would squash the commits

How do you do that ? Kind of new to git and I didn't fully understand how can I squash multiple commits on a PR, thanks.

**Comment by jonringer** [comment]
@ImExtends 

to squash the last 3 commits into the 4th, you can do:
```
git reset HEAD^^^
git add .
git commit --amend --no-edit
git push ... ... --force
```

This will make your changes appear as 1 commit, instead of 4

**Comment by ImExtends** [comment]
> @ImExtends
> 
> to squash the last 3 commits into the 4th, you can do:
> 
> ```
> git reset HEAD^^^
> git add .
> git commit --amend --no-edit
> git push ... ... --force
> ```
> 
> This will make your changes appear as 1 commit, instead of 4

Thanks a lot, better now ?

**Comment by jonringer** [comment]
yea, a lot cleaner, thanks :)

File: README.md [hunk 1]
```diff
@@ -128,6 +129,7 @@ See code for all available configurations.
 | [Tuxedo InfinityBook v4][]        | `<nixos-hardware/tuxedo/infinitybook/v4>`          |
 
 [Acer Aspire 4810T]: acer/aspire/4810t
+[Asus TUF FX504GD]: asus/fx504gd
```
Comment (Mic92) [inline README.md:132]: It's not yet documented but can you also add it here? https://github.com/NixOS/nixos-hardware/blob/master/flake.nix

Comment (ImExtends) [reply README.md:132]: Is it good ? I kind of think I messed up a little bit by trying to commit and squash, tell me if I need to change something.

Comment (Mic92) [reply README.md:132]: I can squash on merge.

File: flake.nix [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (Mic92) [inline flake.nix:1]: This looks indeed weird. I have a look later.

Comment (ImExtends) [reply flake.nix:1]: Thanks, send a message here if I need to do something.

**Comment by Mic92** [comment]
merged in https://github.com/NixOS/nixos-hardware/pull/185

**Pull request closed without merging**